files/default/github_pr_comment_trigger.rb

```diff
@@ -0,0 +1,109 @@
+#!/opt/chef/embedded/bin/ruby
+
+# Takes a JSON payload from a Github PR issue webhook from stdin, reads the
+# comment, and if it matches `!bump (major|minor|patch)`, merges the PR, bumps
+# the Chef cookbook's metadata.rb, creates a version tag, and pushes up.
+
+require 'git'
+require 'json'
+require 'octokit'
+require 'pp'
+
+METADATA_FILE = 'metadata.rb'.freeze
+CHANGELOG_FILE = 'CHANGELOG.md'.freeze
+COMMAND = '!bump'.freeze
+LEVELS = {
+  'major' => 0,
+  'minor' => 1,
+  'patch' => 2
+}.freeze
+
+# Given a version string of the form 'x.x.x' and a level 0, 1, or 2, increments
+# the specified level and returns the new version string.
+def inc_version(v, level)
+  v = v.split('.')
+  v[level] = v[level].to_i.next.to_s
+  v.join('.')
+end
+
+d = JSON.load(STDIN.read)
+
+# Check if we got a valid request and get the bump level
+comment = d.fetch('comment', {}).fetch('body', '')
+match = comment.match(/^#{COMMAND} (#{LEVELS.keys.join('|')})$/, 2)
+if match.nil?
+  # Exit because it wasn't a valid bump request
+  exit 0
+end
+level = match[1]
+
+# Make sure the issue is a PR
+unless d['issue'].key?('pull_request')
+  puts 'Error: Cannot merge issue; can only merge PRs.'
+  exit 1
```

Comment:
Ruby has a nice `abort` command which takes a string and dumps it to stderr and exits 1.

http://ruby-doc.org/core-2.3.0/Kernel.html#method-i-abort